test/reduce_test.cpp

```diff
@@ -41,10 +41,6 @@
 
 #include "cpu_reduce_util.hpp"
 
-/// Not reproducible with ROCm 4.0 and 4.1.
```

Comment:
[Note] You can also remove config.h at line 26, it is not necessary anymore. See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/774/commits/58831650790227e9fc6f84c9365fd9a66c413ac7